kyo-prelude/shared/src/main/scala/kyo2/Aspect.scala

```diff
@@ -0,0 +1,65 @@
+package kyo2
+
+object Aspect:
```

Comment:
I've kept the same APIs as the original effects in `kyo-core`, only adapting the internals if needed. For example, in this case no changes were needed.